src/main/scenes/svg.tsx

```diff
@@ -212,19 +214,61 @@ export class Svg extends Component<Props, State> {
             </defs>
             {elements.map((element, index) => {
               const ElementComponent = Components[element.type as UMLElementType | UMLRelationshipType];
-              return (
-                <svg
-                  x={element.bounds.x - translationFactor().minX}
-                  y={element.bounds.y - translationFactor().minY}
-                  width={element.bounds.width}
-                  height={element.bounds.height}
-                  key={element.id}
-                  className={element.name ? element.name.replace(/[<>]/g, '') : ''}
-                  fill={element.fillColor || theme.color.background}
-                >
-                  <ElementComponent key={index} element={element} scale={this.props.options?.scale || 1.0} />
-                </svg>
-              );
+              switch (ElementComponent) {
+                case UMLClassifierComponent:
+                  // If the ElementComponent is of type UMLClassifierComponent, create an array of all members (attributes and methods) for that component. 
+                  // Unlike other components, the UMLClassifierComponent needs its members to be children within the component to avoid border rendering issues.
+                  const members = elements.filter((member) => member.owner === element.id);
+                  return (
+                    <svg
+                      x={element.bounds.x}
+                      y={element.bounds.y}
+                      width={element.bounds.width}
+                      height={element.bounds.height}
+                      key={element.id}
+                      className={element.name ? element.name.replace(/[<>]/g, '') : ''}
```

Comment:
This happens 3 times within this component, might be worth it to use a method `convertElementNameToClass` or something similar to improve readability and reduce code duplication?

Reply:
Good idea! I created a method for the duplicate code.